src/reference-manual/transforms.qmd

```diff
@@ -499,7 +495,7 @@ S_N = 0
 $$
 and
 $$
-y_N = -x_{N + 1} \cdot \frac{\sqrt{N \cdot (N + 1)}{N}}.
+y_N = -x_{N + 1} \cdot \frac{\sqrt{N \cdot (N + 1)}}{N}.
```

Comment:
[optional]

You can simplify with 

$\displaystyle \sqrt{1 + \frac{1}{N}} \ = \ \frac{\sqrt{N \cdot (N + 1)}}{N}$.   

With $N \in \mathbb{N}$ it seems like the simpler form would also be more stable, but my intuitions for numerics are pretty bad.